R/cnd-abort.R

```diff
@@ -665,17 +665,29 @@ utils::globalVariables(".internal")
 
 footer_internal <- function(env) {
   top <- topenv(env)
+  url_line <- NULL
+
   if (is_namespace(top)) {
-    pkg <- sprintf(" in the %s package", ns_env_name(top))
+    pkg <- ns_env_name(top)
+    url <- pkg_url_bug(pkg)
+    pkg_line <- sprintf(
+      "This is an internal error that was detected in the %s package.",
+      format_pkg(pkg)
+    )
+
+    url <- pkg_url_bug(pkg)
+    if (!is_null(url)) {
+      url_line <- sprintf(
+        "Please report it at %s with a %s and the full backtrace.",
+        format_url(url),
+        format_href("reprex", "https://reprex.tidyverse.org/")
```

Comment:
```suggestion
        format_href("reprex", "https://tidyverse.org/help")
```

I actually send people to this link instead https://tidyverse.org/help, I find it to be the clearest example of what we actually want them to do - up to you about which link to use

Reply:
oh perfect